app/controllers/admin/comparisons/footnotes_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Admin::Comparisons
+  class FootnotesController < AdminController
+    include LocaleHelper
+
+    load_and_authorize_resource :footnote, class: 'Comparison::Footnote'
+
+    def index
```

Comment:
Suggest having an explicit ordering here to keep index tidy. Sort by the key?

Reply:
Good idea, thanks